src/main/java/seedu/address/model/client/Client.java

```diff
@@ -133,6 +135,16 @@ public boolean isSameClient(Client otherClient) {
                 && otherClient.getClientName().equals(getClientName());
     }
 
+    /**
+     * Returns true if client is valid and exists.
+     */
```

Comment:
May I ask why the `isEmpty` method does not work here? (Unless there is another check that is being considered for validity, apart from `Client` object being an emptyClient?)